src/mcp/server/fastmcp/tools/tool_manager.py

```diff
@@ -19,8 +19,16 @@
 class ToolManager:
     """Manages FastMCP tools."""
 
-    def __init__(self, warn_on_duplicate_tools: bool = True):
+    def __init__(
+        self, tools: list[Tool] | None = None, warn_on_duplicate_tools: bool = True
```

Comment:
This is a breaking change, let's use a `*` and put it after warn_on_duplicate_tools